src/components/pages/collectRecordFormPages/BenthicPhotoQuadratForm/BenthicPhotoQuadratForm.js

```diff
@@ -208,22 +211,39 @@ const BenthicPhotoQuadratForm = ({ isNewRecord = true }) => {
       updateBenthicAttributeOptionsStateWithOfflineStorageData,
     ],
   )
+
   const PartiallyAppliedBenthicPhotoQuadratObservationsTable = useCallback(
     (props) => {
-      // the conditional here makes tests happy
-      if (benthicAttributeSelectOptions.length) {
+      const manualObservationRecordsLength =
+        collectRecordBeingEdited?.data?.obs_benthic_photo_quadrats.length
+
+      if (!observationTableType && !manualObservationRecordsLength) {
+        return <SampleUnitInputSelector setObservationTableType={setObservationTableType} />
+      } else if (observationTableType === 'manual-input' || manualObservationRecordsLength) {
+        return (
+          <>
+            <BenthicPhotoQuadratObservationTable
+              benthicAttributeSelectOptions={benthicAttributeSelectOptions}
+              {...props}
+            />
+          </>
+        )
+      } else if (observationTableType === 'image-classification') {
         return (
-          <BenthicPhotoQuadratObservationTable
-            benthicAttributeSelectOptions={benthicAttributeSelectOptions}
-            {...props}
-          />
+          <>
+            <ImageClassificationTable />
+          </>
         )
```

Comment:
The fragments aren't necessary here since you're only returning a single element in both cases

Reply:
good point! will remove